.dockerignore

```diff
@@ -0,0 +1,5 @@
+node_modules
```

Comment:
I think the ignored node_modules is the reason why it did not see the tsc. I am not sure.